packages/vue/src/components/FModal/FModal.vue

```diff
@@ -119,6 +119,19 @@ export default defineComponent({
                 return sizes.includes(value);
             },
         },
+        /**
+         * Default behavior is that the modal will restore focus to previous element once closed.
+         * "on" (default) - component will set focus both when opened and closed
```

Comment:
Nitpick: kanske köra en markdown lista, ser lite snyggare ut i dokumentationen:

```suggestion
         * - "on" (default) - component will set focus both when opened and closed
```

Även kanske formatera det med backticks?

```suggestion
         * - `"on"` (default) - component will set focus both when opened and closed
```